### DISCUSSION TITLE
Frage zum Bunkersilo-Modus - Nutzung von Siloverteilern im Fahrsilo

### DISCUSSION THREAD

**Discussion by TMac063** [Courseplay/Courseplay_FS22#2723] [Q&A]
Moin zusammen,
ich spiele mit einem Kollegen im MP Modus die NF Marsch 4-Fach Map.
Nun möchten wir mit CP und AD ein Fahrsilo befüllen und verdichten. Mir ist aufgefallen, dass der Bunkersilo Modus
erst angezeigt wird, wenn ich einen Schiebeschild hinten am Trecker montiert habe. Bei der Suche nach einem ent-
sprechenden Tutorial bin ich auf das von TestArea auf YT gestossen. Dort wird für das Fahrsilo ein Verteiler und eine Walze
 genutzt, da im Fahrsilo ja ein Schiebeschild ja eher schwierig zu nutzen ist. Der CP Bunkersilo Modus funktioniert dort einwandfrei. Bei mir im MP Modus wird jedoch kein Bunkersilo Modus angezeigt, wenn ich einen Siloverteiler (Holaras Junbo) am Trecker montiert habe.
 Bin irgendwo falsch abgebogen?
 CP Version 7.2.1.6
 LS 22 1.12.0.0
 Vielen Dank im Voraus 
 McFarmer63

**Comment by Tensuko** [comment]
Bitte mal die neueste Version auf den Server spielen.
An der stelle wo "nahester" WP steht, sollte sich der "modus" zum Bunker Silo Modus durchschalten.

Ist das nicht der Fall, müssen wir as noch mal anpassen.
Alternativ eine Kombi von Verteiler und Rolle benutzen, müsste gehen.

**Comment by TMac063** [comment]
Hi Tensuko,
Danke für den Tip. Haben wir gemacht und der Silomodus funktioniert jetzt.
Habe aber eine weitere Frage.
Wieviele Platz braucht CP um die Enden von Silobunkern herum
, um richtig arbeiten zu können?
Hintergrund der Frage ist, dass bei unserer Version der NF-4-fach Map (eine Variante wo durch einen Modder,  die Karte liebevoll ausstaffiert und bereits mit allen unterstützten Produktionen versehen ist), die Fahrsilos so platziert sind, dass der CP Helfer im Silo-Modus sich beim Rangieren immer in den Zäunen festfährt(s. Bilder).
Gibt es dort eine Lösung?
Nochmals vielen lieben Dank im Voraus für Eure Mühen.
Grüße
McFarmer63
![BGA Süd Fahrsilo 1 2_a](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/37bf1274-58f9-42ed-9545-297f6e63e8dc)
![BGA Süd Fahrsilo 1 2_b](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/adf1b8eb-d680-4d6f-9f6f-a5905564d211)
![BGA Süd Fahrsilo Stuck](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/8381de12-5ba2-4ce2-873a-36627aad8545)

**Reply by Tensuko** [reply]
Das jetzt so sagen ist schwer, aber offenbar mehr als ihr habt.
Du kannst im SP mal einen Spielstand erstellen und einfach einen Traktor ohne alles (sollte so gehen) in dem Modus starten und dabei in den Fahrzeugeinstellungen ganz untden den debugmodus einschalten.
Da siehst du dann welchen Platz CP zum rangieren benötigen.
Das liegt einfach daran das CP bei den Wänden ja möglichst gerade fahren muss um nicht in der Wand hängen zu bleiben.
Eventuell kann man das mal etwas anpassen, dass er dann zwar extra fahrten macht, aber dafür nicht so viel Platz braucht (vor zurück vor zurück). Müssen wir mal schauen.

**Comment by TMac063** [comment]
Hi Tensuko,
habe Deinen Vorschlag umgesetzt. Wenn ich die Bilder richtig deute, sind die rot umrandeten Flächen an den Enden des Fahrsilos, die Bereiche, die CP zur Berechnung der Spurwechsel und die An- und Abfahrten von zur Parkposition nutzt?
Wie man an den Bildern sehen kann, sind diese Flächen für das aktuell genutzte Fahrsilo so groß, dass es dort jede Menge Möglichkeiten für den CP Helfer gibt, sich festzufahren. Ich wage sogar die Behauptung, dass das Fahrsilo nebendran gar nicht mit dem Silomodus genutzt werden kann, da an den Enden des Silos der Platz für CP von der Map her nicht ausreicht.
Habe ich das so richtig verstanden?
Nochmals vielen lieben Dank für Deine Zeit und Deine Mühen
Grüße
McFarmer63

![CP Silomodus-1](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/dd29419b-36f5-4ab9-9664-bdd8702accf2)
![CP Silomodus-2](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/bb52eac1-59da-417f-a7dc-d1e9534624d8)

**Comment by Tensuko** [comment]
Die roten Flächen sind der Bereich in dem er nach abfahrern schaut um dann zum parkpunkt zu fahren.
Du müsstest ihn laufen lassen und die blauen linien angucken.

**Comment by TMac063** [comment]
Hi Tensuko,
vielen Dank für die Antwort und Deine Geduld.
Also, wenn ich das richtig verstanden habe, meintest Du den Front- bzw. Backmarker (s. Kreis), richtig?
Mir ist zudem aufgefallen, dass sich der CP-Helfer, bislang nur bei der Rückwärtsfahrt festgefahren hat und das auf beiden Seiten dieses Silos (wie man im Bild Debug 3 sieht , liegt der Backmarker deutlich hinter dem Zaun der BGA). Bei der Vorwärtsfahrt gabs bislang keine Probleme (zumindest bei diesem Silo - das erste Silo auf dieser Karte, wo wir den CP-Silomodus nutzen).
Ich hoffe, das hilft weiter.
Nochmals vielen lieben Dank für Deine Mühen und ein schönes Wochenende.
Grüße
McFarmer63
![CP Silomodus - Debug 3](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/ea76ac35-869d-42b5-b9f1-8c526109ef78)
![CP Silomodus - Debug 4](https://github.com/Courseplay/Courseplay_FS22/assets/62108471/4bc60019-da91-4499-bc47-5410e489ce8e)

**Comment by Tensuko** [comment]
Diese blaue Linie:
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/2a68dc87-9e71-4f8c-806c-d8d45d6b1f2a)
